scripts/link-data.sh

```diff
@@ -79,6 +80,9 @@ link_locs=(
   scRNA-seq-advanced/data/PBMC-TotalSeqB/raw_feature_bc_matrix
   scRNA-seq-advanced/data/PBMC-TotalSeqB/normalized/PBMC_TotalSeqB_normalized_sce.rds
   scRNA-seq-advanced/data/glioblastoma/raw_feature_bc_matrix
+  scRNA-seq-advanced/data/SCPCP000005-samples/raw
+  scRNA-seq-advanced/data/SCPCP000005-samples/processed
+  scRNA-seq-advanced/data/SCPCP000005-samples/annotations
```

Comment:
Do we need users to have access to the `raw` and `annotation` directories? If these are used only for the prep scripts, they can be left off of this linking script.